src/components/FeatureViews/Scheduler/SessionFilters.js

```diff
@@ -61,20 +63,22 @@ function SessionFilters({
 					// styles={customStyles}
 					placeholder={"Filter Instructor..."}
 					value={InstructorValue}
-					options={InstructorOptions}
+					options={InstructorOptions || []}
```

Comment:
Set the default value of InstructorOptions to be the empty array `[]` instead of doing that here.

Reply:
I did this: 
```
    const instructorOptions = useMemo(() => {
        if (!instructors) return []
        Object.entries(instructors).map(
            ([instructorID, instructor]) => ({
                "label": instructor.name,
                "value": instructorID,
            })

        )
    }, [instructors]);
```

Reply:
Sounds good, is it included in this PR? Don't forget to return the mapped array as well.